src/sentry/stacktraces/processing.py

```diff
@@ -177,22 +177,36 @@ def preprocess_step(self, processing_task):
 
 
 def find_stacktraces_in_data(
-    data: NodeData, include_raw: bool = False, with_exceptions: bool = False
+    data: NodeData, include_raw: bool = False, include_empty_exceptions: bool = False
 ) -> list[StacktraceInfo]:
-    """Finds all stacktraces in a given data blob and returns it
-    together with some meta information.
+    """
+    Finds all stacktraces in a given data blob and returns them together with some meta information.
 
     If `include_raw` is True, then also raw stacktraces are included.
 
-    If `with_exceptions` is set to `True` then stacktraces of the exception
-    are always included and the `is_exception` flag is set on that stack
-    info object.
+    If `include_empty_exceptions` is set to `True` then null/empty stacktraces and stacktraces with
+    no or only null/empty frames are included (where they otherwise would not be), with the
+    `is_exception` flag is set on their `StacktraceInfo` object.
     """
     rv = []
 
-    def _append_stacktrace(stacktrace: Any, container: Any, is_exception: bool = False) -> None:
-        if not is_exception and (not stacktrace or not get_path(stacktrace, "frames", filter=True)):
-            return
+    def _append_stacktrace(
+        stacktrace: Any,
+        # The entry in `exception.values` or `threads.values` containing the `stacktrace` attribute,
+        # or None for top-level stacktraces
+        container: Any = None,
+        # Whether or not the container is from `exception.values`
+        is_exception: bool = False,
+        # Prevent skipping empty/null stacktraces from `exception.values` (other empty/null
+        # stacktraces are always skipped)
+        include_empty_exceptions: bool = False,
+    ) -> None:
+        if is_exception and include_empty_exceptions:
+            # win-fast bypass of null/empty check
+            pass
+        else:
+            if not stacktrace or not get_path(stacktrace, "frames", filter=True):
```

Comment:
You can now use this instead of `get_path`:
https://github.com/getsentry/sentry/blob/3f3b47f263d9fbf6e39993c566b30974bf20b2d6/src/sentry/stacktraces/processing.py#L39-L43

You will need to look at the updated code here:
https://github.com/getsentry/sentry/blob/3f3b47f263d9fbf6e39993c566b30974bf20b2d6/src/sentry/stacktraces/processing.py#L204-L208

Reply:
Yup, I'll rebase on top of your changes, thanks.